meeshkan/gen/generator.py

```diff
@@ -0,0 +1,669 @@
+from functools import reduce
+import lenses
+from lenses.optics.base import Prism
+lens = lenses.lens
+import jsonschema
+import json
+import re
+from http_types import Request
+from typing import cast, Sequence, Tuple, TypeVar, Callable, Optional, Mapping, Union, Any
+from openapi_typed import OpenAPIObject, Response, RequestBody, Header, Operation, Parameter, Components, Reference, Schema, PathItem
+from urllib.parse import urlparse
+
+C = TypeVar('C') 
+D = TypeVar('D')
+S = TypeVar('S')
+T = TypeVar('T')
+W = TypeVar('W')
+X = TypeVar('X')
+Y = TypeVar('Y')
+Z = TypeVar('Z')
+
+def pp(c: C) -> C:
+    print("LOOK", c)
```

Comment:
`from ..logger import get as getLogger; logger = getLogger(__name__); logger.info(...)` maybe instead of print?